structs/src/strg.rs

```diff
@@ -37,6 +37,68 @@ pub struct Strg<'r>
 
 impl<'r> Strg<'r>
 {
+    fn is_jap_version(languages: Vec<FourCC>) -> bool
```

Comment:
This abbreviation of "Japanese" is actually a racial slur whose history can be traced back to war crimes

jpn/jp/j are all fine alternatives

Reply:
I didn't know that

Reply:
btw that means in the dol symbol table we should change to jpn instead of jap?

Reply:
If it's something retro did no, if it's something we did, then probably yeah...

Reply:
I mean check in generated/dol_symbol_table folder